web/xfa_layer_builder.css

```diff
@@ -17,6 +17,37 @@
   --unfocused-field-background: url("data:image/svg+xml;charset=UTF-8,<svg width='1px' height='1px' xmlns='http://www.w3.org/2000/svg'><rect width='100%' height='100%' style='fill:rgba(0, 54, 255, 0.13);'/></svg>");
 }
 
+.xfaLayer .highlight {
+  margin: -1px;
+  padding: 1px;
+  background-color: #efcbed;
```

Comment:
In our existing CSS files, we've tried to settle of *one* colour format; please see https://github.com/mozilla/pdf.js/pull/11447.

Hence can you please use the rgba-format here, and below, instead?